scripts/dockutil

```diff
@@ -403,7 +401,9 @@ def writePlist(pl, plist_path):
     # get a tempfile path for writing our plist
     plist_import_path = tempfile.mktemp(dir='/tmp')
     # Write the plist to our temporary plist for importing because defaults can't import from a pipe (yet)
-    plistlib.writePlist(pl, plist_import_path)
```

Comment:
I might want to check if method exists if plistlib has changed and we want to support either.

Reply:
It doesn't. Python 3.4 and higher only :/